packages/yourloops/components/chat/chat-widget.tsx

```diff
@@ -0,0 +1,231 @@
+import SentimentSatisfiedOutlinedIcon from "@material-ui/icons/SentimentSatisfiedOutlined";
+import EmailOutlinedIcon from "@material-ui/icons/EmailOutlined";
+import Card from "@material-ui/core/Card";
+import ChatMessage from "./chat-message";
+import { getChatMessages, sendChatMessage } from "../../lib/chat/api";
+import { useAuth } from "../../lib/auth";
+import { IMessage } from "../../models/chat";
+
+const chatWidgetStyles = makeStyles((theme: Theme) => {
+  return {
+    chatWidget: {
+      width: "300px",
+      height: "500px",
+    },
+    chatWidgetHeader: {
+      padding: theme.spacing(1),
+      display: "flex",
+      alignItems: "center",
+      background: "#035271",
+      color: "black",
+      width: "100%",
+      height: "40px",
+    },
+    chatWidgetHeaderText: {
+      padding: theme.spacing(1),
+    },
+    iconButton: {
+      "backgroundColor": "white",
+      "border": "none",
+      "& :hover": {
+        cursor: "pointer",
+      },
+    },
+    chatWidgetContent: {
+      background: "white",
+      width: "100%",
+      height: "420px",
+      overflow: "auto",
+    },
+    chatWidgetFooter: {
+      display: "flex",
+      alignItems: "center",
+      borderTop: "1px solid #E9E9E9",
+      background: "white",
+      borderRadius: "0px 0px 12px 12px",
+      width: "100%",
+      height: "40px",
+      paddingTop: theme.spacing(1),
+      paddingBottom: theme.spacing(1),
+    },
+    chatWidgetInput: {
+      "fontFamily": "Roboto",
+      "fontStyle": "normal",
+      "marginLeft": theme.spacing(1),
+      "display": "block",
+      "width": "90%",
+      "maxWidth": "90%",
+      "overflow": "hidden",
+      "resize": "none",
+      "height": "24px",
+      "maxHeight": "60px",
+      "lineHeight": "20px",
+      "border": "1px solid white",
+      "&:focus" : {
+        outlineColor: "#F6F6F6",
+      },
+    },
+    emojiPicker: {
+      display: "none",
+    },
+    icon: {
+      margin: theme.spacing(1),
+    },
+  };
+}, { name: "ylp-chat-widget" });
+
+export interface Message {text: string, isMine: boolean}
+export interface ChatWidgetProps {
+  patientId: string;
+  teamId: string;
+  userId: string
+  role: string
+}
+
+function ChatWidget(props: ChatWidgetProps): JSX.Element {
+  const { patientId, teamId, userId, role } = props;
+  const authHook = useAuth();
+  const [showPicker, setShowPicker] = useState(false);
+  const [inputText, setInputText] = useState("");
+  const [messages, setMessages] = useState<IMessage[]>([]);
+  const content = useRef<HTMLDivElement>(null);
+  const input = useRef<HTMLTextAreaElement>(null);
+  const footer = useRef<HTMLDivElement>(null);
+  const classes = chatWidgetStyles();
+
+  useEffect(() => {
+    content.current?.lastElementChild?.scrollIntoView();
+  }, [messages]);
+
+  useEffect(() => {
+    async function fetchMessages() {
+      const session = authHook.session();
+      if (session !== null) {
+        console.log(patientId);
```

Comment:
Did you forget to delete these console.log?

Reply:
fixed